in_toto/model.go

```diff
@@ -31,6 +35,24 @@ type Key struct {
 	Scheme              string   `json:"scheme"`
 }
 
+func validateKeyId(keyId string) error {
+	keyIdFormatCheck, _ := regexp.MatchString("^[a-fA-F0-9]+$", keyId)
+	if !keyIdFormatCheck {
+		return fmt.Errorf("'Key ID' has invalid format")
+	}
+	return nil
+}
+
+func validatePubKey(key Key) error {
```

Comment:
Shouldn't `validatePubKey` call `validateKeyId` on its `KeyId`?

Reply:
Please also note that key schemas in the reference implementation adds a bunch of more restrictions.

Currently we only support rsa keys, so it's probably worth having at least a `validatePublicRSAKey` (see `securesystemslib.formats.RSAKEY_SCHEMA`)

Reply:
This has been generalized to `validateHexSchema` + `validateRSAPubKey` now.